.github/workflows/welcome-first-time-contributors.yml

```diff
@@ -0,0 +1,37 @@
+name: Welcome first time contributors
+
+on:
+  pull_request_target:
+    types:
+      - opened
+  issues:
+    types:
+      - opened
+
+jobs:
+  welcome:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/first-interaction@v1
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+        issue-message: |
+          Hello there ${{ github.actor }} 👋
+
+          Welcome to tsoa !!💖🥳
+
+          Thank you and congratulations 🎉 for opening your very first issue in this project. tsoa fosters an open and welcoming environment for all our contributors.🌸
+
+          Incase you want to claim this issue, please comment down below! We will try to get back to you as soon as we can.👀
+
+          👩‍💻 If you have any interesting ideas, just open an issue. We would love to hear you and engage in discussions.
+
```

Comment:
I think a bot message for every first time someone opens an issue may be too noisy. Let's focus on a good PR message instead.